circuit-test/src/commonMain/kotlin/com/slack/circuit/test/FakeNavigator.kt

```diff
@@ -37,6 +37,10 @@ public class FakeNavigator : Navigator {
     return null
   }
 
+  override fun peek(): Screen? {
+    error("peek() is not supported in FakeNavigator!")
```

Comment:
I'm revisiting this in another PR, so we can leave this like so. I do think we can make this fake more realistic and track screens, and probably consider dropping turbine from it